### ISSUE TITLE
Unable to deposit stack using Trezor One

### ISSUE THREAD

**Issue by resherwni** [thetatoken/guardian-mainnet-guide#14]
I'm unable to deposit a stake to a Guardian node using a Trezor One hardware wallet. It just goes in to a continuous loop continuously asking me to reconfirm the transaction and then giving me a nonsensical error message.

I've recorded a screen cast showing the steps I'm going through. You can see warnings and error messages during this. See

https://www.dropbox.com/s/ffz0eke5lh1zt5l/Theta%20Token%20Wallet%20Unable%20to%20Deposit%20Stake.mp4?dl=0

**Comment by jieyilong** [comment]
Thanks for the screen cast. The error means you need a small amount of TFUEL for the transaction fee. Do you mind transfer 1 TFUEL to your wallet and try again? Thanks.

**Comment by resherwni** [comment]
That part wasn't mentioned in the instructions.

**Comment by jieyilong** [comment]
Thanks I'll add it to the instructions

**Comment by jackdublin** [comment]
hello.  I am an extreme novice but trying to stake my Theta. I succesfully stored my Theta onto a Model T.  Today, I attempted to stake. When I connected to my Mac, was prompted to update firmware, which I did succesfully. When I connected to Theta wallet, and when attempted staking, pop up warned that Trezor bridge was not up to date with latest Trezor firmware, and to ask wallet developer to update Trezor bridge.  Screen would just spin wheel on wallet page, while pop up would continue me through warning pop up, click proceed button, back to wallet page "deposit and stake", etc.  when i started from scratch again, I thought it made a break through, screen said "follow instructions on Trezor screen".  But Trezor screen had some jibberish message "wei to " followed by many zeros. And option to click red to cancel, or green to proceed.  I clicked red, I didnt trust what i was seeing. Please advise.  Thank you

**Comment by RichX720** [comment]
> ![Screenshot_2021-03-01 TrezorConnect Trezor](https://user-images.githubusercontent.com/79101570/109525337-6e021e80-7a7f-11eb-8544-c4d120f6fe3d.png)
> ![Screenshot_2021-03-01 TrezorConnect Trezor(1)](https://user-images.githubusercontent.com/79101570/109525339-6e9ab500-7a7f-11eb-8eea-2c1b3144ae60.png)
> ![Screenshot_2021-03-01 TrezorConnect Trezor(2)](https://user-images.githubusercontent.com/79101570/109525341-6e9ab500-7a7f-11eb-86cb-209e7950ea76.png)
> 
> hello. I am an extreme novice but trying to stake my Theta. I succesfully stored my Theta onto a Model T. Today, I attempted to stake. When I connected to my Mac, was prompted to update firmware, which I did succesfully. When I connected to Theta wallet, and when attempted staking, pop up warned that Trezor bridge was not up to date with latest Trezor firmware, and to ask wallet developer to update Trezor bridge. Screen would just spin wheel on wallet page, while pop up would continue me through warning pop up, click proceed button, back to wallet page "deposit and stake", etc. when i started from scratch again, I thought it made a break through, screen said "follow instructions on Trezor screen". But Trezor screen had some jibberish message "wei to " followed by many zeros. And option to click red to cancel, or green to proceed. I clicked red, I didnt trust what i was seeing. Please advise. Thank you
> PS I just included screen shots of my latest attempt. I did not get messages to go to Trezor screen, however, still got messages and requests that are not in staking instruction pages, illustrated in screen shots. Help. Thanks

I have this same issue with my TREZOR T as well on a windows PC. Please advise??

**Comment by Garitty7612** [comment]
@RichX720 when did you notice the issue ?

**Comment by jackdublin** [comment]
First time I tried staking. Like a month ago. After trying a couple times, and doing some simpleswap transactions, the Trezor message of "might not be up to date" are acceptable. I get them when logging on. But trying to stake, getting the message to "follow instructions on Trezor screen", and getting this gibberish "wei to", zero's, and the red and green options, suggesting approve or don't approve, but it isnt clear. It should be crystal clear. HELP!!!

**Comment by Garitty7612** [comment]
@jackdublin Has your issue been resolved ?

**Comment by jackdublin** [comment]
No. Im not staked. I don't know why I get that "wei to", etc message when trying to stake. None of the how to vids have that screen image. So I've been unplugging and not pressing red or green button on Trezor, because its not in English. When every other transaction has been in English. Green: yes, or accept. Red: no, or decline. No one has replied that that screen is correct or has been seen before.

**Comment by Garitty7612** [comment]
Okay I had a similar issue to that. I had to contact the theta network support on telegram. You can contact Theta Network Support on Telegram via the weblink https://t.me/thetanetworking. They'll be happy to resolve your issue. Cheers.

**Comment by jackdublin** [comment]
Thanks. I emailed to Theta support. No joy. At least not yet. Go Theta!

**Comment by Garitty7612** [comment]
Okay. You get instant replies on the telegram support. I’ll recommend that for you. Theta Network Support on Telegram via the weblink https://t.me/thetanetworking.

**Comment by jackdublin** [comment]
Will definitely try. Thanks again.

**Comment by Garitty7612** [comment]
Do let me know whatever your outcome is. I’m glad I could help.
Do give it a try

**Comment by Garitty7612** [comment]
@jackdublin Has your issue been resolved ?

**Comment by RichX720** [comment]
Still no fix as yet..

**Comment by RichX720** [comment]
Telegram is closed for Theta.. you hace to use Discord now..

**Comment by RichX720** [comment]
See Step by Step issue:

![step01](https://user-images.githubusercontent.com/70340479/111704888-21b92b80-8840-11eb-89fd-1dbfd658fccd.png)
![step02](https://user-images.githubusercontent.com/70340479/111704889-2251c200-8840-11eb-845a-a0045e255fca.png)
![step03](https://user-images.githubusercontent.com/70340479/111704890-2251c200-8840-11eb-8dbe-63fc4a2e412b.png)
![step04](https://user-images.githubusercontent.com/70340479/111704891-22ea5880-8840-11eb-8916-4190482c3244.png)
![step05](https://user-images.githubusercontent.com/70340479/111704892-22ea5880-8840-11eb-8cb3-d54395c0d3aa.png)
![step06](https://user-images.githubusercontent.com/70340479/111704893-22ea5880-8840-11eb-86ea-20b36dc6998b.png)
![step07](https://user-images.githubusercontent.com/70340479/111704894-2382ef00-8840-11eb-9687-4be06bcb6e1b.png)
![step08](https://user-images.githubusercontent.com/70340479/111704895-2382ef00-8840-11eb-98f0-7304980d03dd.png)
![step09](https://user-images.githubusercontent.com/70340479/111704897-241b8580-8840-11eb-9ca9-bd9f2946034d.png)
![step10](https://user-images.githubusercontent.com/70340479/111704898-241b8580-8840-11eb-88ed-408272129b3d.png)
![step11](https://user-images.githubusercontent.com/70340479/111704900-241b8580-8840-11eb-9800-95626e141a05.png)
![step12](https://user-images.githubusercontent.com/70340479/111704903-24b41c00-8840-11eb-90d5-e5bcaffe7916.png)
![step13](https://user-images.githubusercontent.com/70340479/111704904-24b41c00-8840-11eb-84ab-89c33d850acc.png)
![step14](https://user-images.githubusercontent.com/70340479/111704907-25e54900-8840-11eb-9485-25c759bcb06a.png)

**Comment by jackdublin** [comment]
Yes! That is the exact screen I get! Does everyone's Trezor get that screen staking Theta at Gpool?

**Comment by RichX720** [comment]
I am staking on my own Guardian Node :(

**Comment by RichX720** [comment]
If I may ask.. where did you buy your device from?

**Comment by jackdublin** [comment]
I bought straight from Trezor. Now if I may ask, are those pics of you staking to your guardian node? Or did you try Gpool?

**Comment by RichX720** [comment]
To the Guardian Node.. trying to atleast. I have not tried gpool but I'm gonna guess its the same... No one in Discord seems to have this issue so we are the 2 out of the group it seems.. I just emailed Trezor fdirect. Will see what comes of it

**Comment by jackdublin** [comment]
Lucky us. So you get that "wei to" screen, and then you don't click anything on Trezor screen, right? Because it looks too weird? I just unplug, and back my way out of screens on the Mac

**Comment by RichX720** [comment]
lol. I hit the X and it goes away and the theta wallet just shows a blank error message..

**Comment by jackdublin** [comment]
Lol. I might have done that too. I dont remember. Im on vaca right now. All i know is that screen looks weird, and thats after computer screen says follow instructions on the device! I'm afraid to hit the green check.

**Comment by RichX720** [comment]
OK.. so I havent been brave enough yet to do this 1000 theta but i tried SEND 1tfuel to my mobile wallet and the SAME odd message showed up to verify.. so with 1tfuel at stake.. I pressed the green Check.. See the screenshots.. At the end.. Checked my balance and 1 TFUEL arrived in my mobile wallet!!! SO it does work.. next is to grow a larger set of tomatos and do it with 1000 theta!

![trezor_1](https://user-images.githubusercontent.com/70340479/111714335-b6785500-8851-11eb-81c1-d98f5d1270f0.jpeg)
![trezor_2](https://user-images.githubusercontent.com/70340479/111714336-b710eb80-8851-11eb-8b86-690f8f6b4c1b.jpeg)
![trezor_3](https://user-images.githubusercontent.com/70340479/111714338-b7a98200-8851-11eb-99c3-78301690fa46.jpeg)

**Comment by RichX720** [comment]
I was explained that something we did during install caused this. Its trying to see an Etherium address but that's not what Theta is so it just shows UNKN... Im not sure if this is because I set mine up with the new Trezor Suite.. I honestly cant remember what options I could have chosen to cause it but they say it all should be cleared up in Q2 when there is hopefully a native APP supporting theta... Im still afraid to send 1000 theta lol

**Comment by RichX720** [comment]
![staked_redacted](https://user-images.githubusercontent.com/70340479/111716529-83849000-8856-11eb-8155-cfdbf24e4957.png)
IT WORKED!

**Comment by jackdublin** [comment]
Wow! You're crazy! I remember updating firmware when prompted, and i felt like thats when it started acting weird. I had already moved Theta to the wallet. Then a different time i went to stake, was asked to update, and it was weird afterwards. IS ANYONE ELSE LISTENING?? THETA LABS, HELLO?!?
